src/main/resources/com/google/cloud/spanner/jdbc/DatabaseMetaData_GetColumns.sql

```diff
@@ -24,6 +24,7 @@ SELECT TABLE_CATALOG AS TABLE_CAT, TABLE_SCHEMA AS TABLE_SCHEM, TABLE_NAME, COLU
     WHEN SPANNER_TYPE = 'INT64' THEN -5
     WHEN SPANNER_TYPE = 'NUMERIC' THEN 2
     WHEN SPANNER_TYPE LIKE 'STRING%' THEN -9
+    WHEN SPANNER_TYPE = 'JSON' THEN -9
```

Comment:
According to the JDBC specification this should always return one of the constants in `java.sql.Types`. That means that we should not return the vendor code of the JSON type here.